ibmvmc/ibmvmc.c

```diff
@@ -82,6 +82,8 @@ static inline void h_free_crq(uint32_t unit_address)
 	do {
 		if (H_IS_LONG_BUSY(rc))
 			msleep(get_longbusy_msecs(rc));
+		else if (rc)
+			msleep(100);
```

Comment:
That is longer than most long busy values.  Really, H_BUSY means retry immediately.  How do other CRQ drivers work?  If we really want a sleep in here, it should be much smaller.  More like 1ms.

Reply:
oh I didnt know H_BUSY meant to retry immediately. If that is the case then it is good as is. Other drivers aren't implemented as well, its just a standard if(rc) msleep(100); without the part you have up there.